main.go

```diff
@@ -83,5 +85,5 @@ func main() {
 	//Websockets
 	e.Any("/socket.io/", controllers.WebsocketHandler())
 
-	e.Logger.Fatal(e.Start("0.0.0.0:1323"))
+	e.Logger.Fatal(e.Start("localhost:1323"))
```

Comment:
is this fine to leave it as localhost? I changed it in development but maybe we should change it back for docker.

Reply:
oh whoops